.github/workflows/test.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install flake8
-          pip install -r requirements.txt
+          pip install -r tests/requirements.txt
```

Comment:
very good thinking! in yellowbrick we have always had two different sets of requirements so that we can test against things like pandas and stuff that aren't necessarily required for the minimal yb installation